src/views/portfolio-page/utils.ts

```diff
@@ -0,0 +1,63 @@
+import { ProjectType } from './types';
+
+const maxNumberOfTabs = 3;
+const otherLabel = 'More';
+const allLabel = 'All';
+
+type GroupedProjectsByLabelType = {
+  projectLabel: string;
+  projectProperties: ProjectType[];
+};
+
+const compareLengthOfProjects = (
+  longerLengthGroupedProjectsByLabel: GroupedProjectsByLabelType,
+  shorterLengthGroupedProjectsByLabel: GroupedProjectsByLabelType,
+): number => {
+  return (
+    shorterLengthGroupedProjectsByLabel.projectProperties.length -
+    longerLengthGroupedProjectsByLabel.projectProperties.length
+  );
+};
+
+const groupBy = (projects: ProjectType[]) => {
```

Comment:
I suggest to name it `groupByProjectLabel` because it has `projectLabel` hardcoded in the function 😄